README.md

```diff
@@ -5,3 +5,30 @@ Web interface for the popular Politech site
 This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app).
 
 There is alot of reading material to be found in their guide [here](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md).
+
+## Yarn
+To start project
+´´´
+yarn start
+´´´
+
+Add a new package as a dependency
```

Comment:
Isn't adding/removing packages basic enough that these commands are cluttering the README?